src/home/marketing/lower-side/paragraph.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@material-ui/core'
+import { ParagraphOne } from 'home/marketing/paragraph-one'
+import { useStyles } from 'home/marketing/upper-side/style'
+
+export const Paragraph = () => <Grid item xs={12} sm={11} md={11} lg={10} className={useStyles().width}>
+  <Box className={useStyles().box} whiteSpace={'normal'} textAlign={'center'}>
+    <Typography paragraph>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore emagna aliqua.
```

Comment:
Let's rename this component to something like `Content`. After all, if we change this paragraph to be a video, par example, we'll have to rename it anyways.

Reply:
Done!